### DISCUSSION TITLE
In the MVVM toolkit, is there a way to add a DisplayNameAttribute to the generated properties?

### DISCUSSION THREAD

**Discussion by Konstantin-tr** [CommunityToolkit/dotnet#739] [Q&A] [answered]
I have an observable property generated from a field, now I need to change its display name but since I only have the field in my class, I can't actually apply the attribute (property only). Is there a way to work around this?

**Comment by Konstantin-tr** [comment]
Actually, I just noticed something similar is possible via the DisplayAttribute, although that one is sadly sealed, making it a bit harder to use. Any other option would still be appreciated.

**Comment by Sergio0694** [comment, marked as answer]
You can use `[property: DisplayName(...)]` on the field to forward any attributes you need on the generated property 🙂

**Reply by Konstantin-tr** [reply]
I thought so, but DisplayNameAttribute has AttributeUsage set to properties only, so you can't add it to the field, that was the issue I faced when originally attempting this.

**Reply by Sergio0694** [reply]
You can if you use `property:`, just try it 😄

**Reply by Konstantin-tr** [reply]
Oh,  that is great, I didn't realize this was possible, thanks a lot 😊
